qc_duplex_bam/README.md

```diff
@@ -68,3 +71,76 @@ optional arguments:
   --sample_group SAMPLE_GROUP
   --maf MAF
 ```
+
+### v2.0 qc_duplex_bam_v2.0.cwl
+
+This sub-workflow was designed for MSK-ACCESS and CMO-CH data. The updated v2.0 of this sub-workflow expects the duplex bam and a single set of bait/target regions as opposed to two in v1.0.
+
+The sub-workflow calculates quality control metrics for duplex BAMs. The main outputs are the following:
+
+1. Targeted capture metrics.
+2. Insert size metrics.
+3. Alignment metrics.
+4. Extracted genotype information used for fingerprinting and contamination estimation.
+5. Genotype metrics to be used for hotspot mutation metrics.
+6. Coverage metrics files to estimate on target coverage. Specific to v2.0.
+
+### Additional Tools used:
```

Comment:
We dont need to write it as additional tools you can explain the intended purpose of the tool as a point

Reply:
@carmelinacharalambous  can you also fix this